test/jasmine/karma.conf.js

```diff
@@ -235,6 +241,8 @@ func.defaultConfig = {
         suppressPassed: true,
         suppressSkipped: false,
         showSpecTiming: false,
+        // use 'karma-fail-fast-reporter' to fail fast w/o conflicting
+        // with other karma plugins
         failFast: false
```

Comment:
So this `failFast` shouldn't ever be used?

Reply:
Setting it to `true` won't break anything, but it won't actually _fail fast_.

Reply:
More on this topic: to make `karma-spec-reporter` _fail fast_, one needs to add it to the _plugins_ list: https://github.com/mlex/karma-spec-reporter#configuration

When doing so, it conflicts with `karma-browserify`. 

There might be a way, to list them (order dependent?) so that they don't conflict, but using [`karma-fail-fast-reporter`](https://github.com/michaelbull/karma-fail-fast-reporter) was easy enough.